src/map/common/lapacksrc/scripts/regen-files.sh

```diff
@@ -121,48 +124,68 @@ then
         ## should verify this is still true
         if [[ "$tar_file" == *.tgz || "$tar_file" == *.tar.gz ]]
         then
-
```

Comment:
nit

Reply:
I added these whitespace changes to remove lines I added. Trying to unify the file.